f/chunks.js

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-function chunks(arr, size) {
+function chunks(arr, size = 1) {
```

Comment:
I disagree on giving `chunks` a default.

Reply:
Okay, I can revert that but then it default to 0:
`chunks([1,2,3,4])` becomes `[ [] ]`

Reply:
Okay, reverted and fixed broken test. 
Also used identify function as default for `unique`, `uniqueLazy` and `intersection`.